InternetHospital.UnitTests/SignUp/RegistrationServiceUnitTests.cs

```diff
@@ -0,0 +1,82 @@
+﻿using AutoFixture;
+using AutoMapper;
+using FluentAssertions;
+using InternetHospital.BusinessLogic.Models;
+using InternetHospital.BusinessLogic.Services;
+using InternetHospital.DataAccess;
+using InternetHospital.DataAccess.Entities;
+using InternetHospital.UnitTests.TestHelpers;
+using Xunit;
+
+namespace InternetHospital.UnitTests.SignUp
+{
+    public class RegistrationServiceUnitTests
+    {
+        public RegistrationServiceUnitTests()
+        {
+            Mapper.Reset();
+            Mapper.Initialize(cfg => cfg.CreateMap<UserRegistrationModel, User>());
+        }
+
+        [Fact]
+        public async void ShouldPatientRegistrate()
+        {
+            const string PATIENT = "Patient";
+
+            var options = DbContextHelper.GetDbOptions(nameof(ShouldPatientRegistrate));
+            var fixture = FixtureHelper.CreateOmitOnRecursionFixture();
+
+            var expectedUser = fixture.Build<User>()
+                .Create();
+
+            var fixtureRegistrationModel = RegistrationHelper.GetRegistrationModel(expectedUser, PATIENT);
+            var fakeUserManager = RegistrationHelper.GetConfiguredUserManager();
+            var fakeRoleManager = RegistrationHelper.GetFakeRoleManager();
+
+            using (var context = new ApplicationContext(options))
+            {
+                var registrationService = new RegistrationService(context, fakeUserManager.Object, fakeRoleManager.Object);
+
+                // act 
+                var res = await registrationService.PatientRegistration(fixtureRegistrationModel);
+
+                // asset
+                res.Email.Should().BeEquivalentTo(expectedUser.Email);
+            }
+        }
+
+        [Fact]
+        public async void ShoulDoctorRegistrate()
+        {
+            const int USER_ID = 2;
+            const string DOCTOR = "Doctor";
```

Comment:
In addition you have const PATIEN and DOCTOR in all methods, maybe its better to remove them into helper class?